tutorials/pretraining-data-curation/red-pajama-v2-curation-tutorial.ipynb

```diff
@@ -0,0 +1,6286 @@
+       "23409     Debian -- Package Search Results -- libisl-dbg   \n",
+       "15108  Debian -- Package Search Results -- libpython3...   \n",
+       "\n",
+       "                                                     url  \n",
+       "5016   https://packages-pkgmirror-csail.debian.org/se...  \n",
+       "4695   https://packages.debian.org/search?lang=uk&sui...  \n",
+       "23690  https://packages.debian.org/search?lang=hu&sui...  \n",
+       "23409  https://packages-pkgmirror-csail.debian.org/se...  \n",
+       "15108  https://packages-pkgmirror-csail.debian.org/se...  "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dup_examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "49b1b00a-501e-4d49-a93b-60a5c8ae87d2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Example duplicate 1\n",
+      "You have searched for packages that names contain lynx-cur in suite(s) stretch-backports-sloppy, all sections, and architecture(s) i386.\n",
+      "\n",
+      "\n",
+      "Example duplicate 2\n",
+      "You have searched for packages that names contain compiz-gtk in suite(s) bookworm, all sections, and architecture(s) hurd-i386.\n",
+      "\n",
+      "\n",
+      "Example duplicate 3\n",
+      "You have searched for packages that names contain compiz-gtk in suite(s) bookworm, all sections, and architecture(s) hurd-i386.\n",
+      "\n",
+      "\n",
+      "Example duplicate 4\n",
+      "You have searched for packages that names contain compiz-gtk in suite(s) bookworm, all sections, and architecture(s) hurd-i386.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Example duplicate 1\\n' + dup_examples.raw_content.iloc[0])\n",
+    "print('\\n\\nExample duplicate 2\\n' + dup_examples.raw_content.iloc[1])\n",
+    "print('\\n\\nExample duplicate 3\\n' + dup_examples.raw_content.iloc[1])\n",
+    "print('\\n\\nExample duplicate 4\\n' + dup_examples.raw_content.iloc[1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c428e09c-5442-4908-8888-4e62994e4c5c",
+   "metadata": {},
+   "source": [
+    "### 6.7 Duplicates Removal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2e01b84-07cc-45a3-9dde-97884d1922a3",
+   "metadata": {},
+   "source": [
+    "Next, we will proceed to remove the duplicates identified from the dataset. We will first change the string ID to `doc_id` and `dataset_id` in the input dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "7cde97c5-cfaa-4096-8d9f-1ffa19c4adb2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reading 37848 files\n"
+     ]
+    }
+   ],
+   "source": [
+    "def convert_str_id_to_int(df, id_column=\"id\"):\n",
```

Comment:
What do you think about putting these in a helper function in `helper.py`? I imagine it would make it simpler for users going through this.

Reply:
done